From c42388cbdf569a2b358c266f2b4a8b75958a3b45 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 29 Sep 2021 14:25:49 +0100
Subject: [PATCH 641/686] defconfigs: Add AD5398 VCM to all Pi defconfigs

Supports AD5398 controlled lens, as found on some 3rd party
OV5647 and IMX219 image sensors.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -874,6 +874,7 @@ CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9281=m
 CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
+CONFIG_VIDEO_AD5398=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_UDL=m
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -889,6 +889,7 @@ CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9281=m
 CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
+CONFIG_VIDEO_AD5398=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_UDL=m
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -867,6 +867,7 @@ CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9281=m
 CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
+CONFIG_VIDEO_AD5398=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_UDL=m
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -894,6 +894,7 @@ CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9281=m
 CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
+CONFIG_VIDEO_AD5398=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_UDL=m
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -815,6 +815,7 @@ CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9281=m
 CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
+CONFIG_VIDEO_AD5398=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_UDL=m
